### ISSUE TITLE
Unable to edit products in non-default site

### ISSUE THREAD

**Issue by zackspear** [craftcms/digital-products#54]
### Description

I'm working on adding localized versions of our site – including our pricing page which utilizes Digital Products. When in the CMS editing the Digital Product and I attempt to swap the site to French for example I get a 404 with the following stack trace:

```
yii\web\NotFoundHttpException: Product Type not found in /home/vagrant/code/unraid/craft-unraid/vendor/craftcms/commerce/src/controllers/ProductsController.php:406
Stack trace:
#0 /home/vagrant/code/unraid/craft-unraid/vendor/craftcms/commerce/src/controllers/ProductsController.php(97): craft\commerce\controllers\ProductsController->_prepEditProductVariables(Array)
#1 [internal function]: craft\commerce\controllers\ProductsController->actionEditProduct('osLicenseKey', 5479, 'french', NULL)
#2 /home/vagrant/code/unraid/craft-unraid/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#3 /home/vagrant/code/unraid/craft-unraid/vendor/yiisoft/yii2/base/Controller.php(180): yii\base\InlineAction->runWithParams(Array)
#4 /home/vagrant/code/unraid/craft-unraid/vendor/craftcms/cms/src/web/Controller.php(189): yii\base\Controller->runAction('edit-product', Array)
#5 /home/vagrant/code/unraid/craft-unraid/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('edit-product', Array)
#6 /home/vagrant/code/unraid/craft-unraid/vendor/craftcms/cms/src/web/Application.php(274): yii\base\Module->runAction('commerce/produc...', Array)
#7 /home/vagrant/code/unraid/craft-unraid/vendor/yiisoft/yii2/web/Application.php(103): craft\web\Application->runAction('commerce/produc...', Array)
#8 /home/vagrant/code/unraid/craft-unraid/vendor/craftcms/cms/src/web/Application.php(259): yii\web\Application->handleRequest(Object(craft\web\Request))
#9 /home/vagrant/code/unraid/craft-unraid/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#10 /home/vagrant/code/unraid/craft-unraid/web/index.php(21): yii\base\Application->run()
#11 {main}
```

### Steps to reproduce

1. Edit a digital product entry
2. Attempt to swap your site
<img width="502" alt="Screen Shot 2020-10-15 at 11 48 10" src="https://user-images.githubusercontent.com/2531682/96173191-b34f0500-0f51-11eb-8c2f-0174c77bfae3.png">

If I attempt to set the site to a non-default site, like French, before trying to edit the entry when I click to edit the French Entry the CMS opens me to the default site – English.
<img width="712" alt="Screen Shot 2020-10-15 at 11 52 56" src="https://user-images.githubusercontent.com/2531682/96173750-8bac6c80-0f52-11eb-9fa3-ee4d1f088a5e.png">

### Additional info

- Craft version: 3.5.12.1
- PHP version: 7.3.4
- Database driver & version: MySQL 5.5.5
- Plugins & versions:
<img width="662" alt="Screen Shot 2020-10-15 at 11 51 09" src="https://user-images.githubusercontent.com/2531682/96173446-16d93280-0f52-11eb-98d3-9bd7eeb4a19d.png">

Happy to provide any additional info and answer any questions. Thanks in advance.

**Comment by tommysvr** [comment]
Thanks @zackspear. Tested and reproduced on a fresh Craft + Digital Products install, and we're looking into it 👍🏻

**Comment by tommysvr** [comment]
Hey @zackspear, would you mind sending a database dump to support@craftcms.com and we'll take a look at it for you.

**Comment by peirix** [comment]
@tommysvr Any progress on this? Have the same issue here.

**Comment by nfourtythree** [comment]
This is fixed with PR #62 and will be included in the next release.
